samples/AspNetCore/Samples.AspNetCore.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net9.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <ManagePackageVersionsCentrally>false</ManagePackageVersionsCentrally>
+  </PropertyGroup>
```

Comment:
Can you move this to the `Common.samples.props`? I would assume that all samples should respect these settings.

I would also encourage to use:
- `<TreatWarningsAsErrors>true</TreatWarningsAsErrors>`